src/compiler/types.ts

```diff
@@ -1587,6 +1587,7 @@ module ts {
         target?: ScriptTarget;
         version?: boolean;
         watch?: boolean;
+        separateCompilation?: boolean;

```

Comment:
i think we need to make use of const enums from an ambient declaration an error while using this module, also consider using of internal modules on the global scope as an error..

also we need to make that available on the command line, as an experimental flag.